jdbc-adapter/integration-test-data/socker.py

```diff
@@ -5,21 +5,30 @@
 import yaml
 import json
 
+DOCKER_NETWORK = 'exasoldb_priv'
+
 def docker_run(config):
     for db, properties in config.items():
         if properties.get('runIntegrationTests', False):
             if 'dockerImage' in properties:
-                cmd = "docker run -d -p {port_map} --name {name} {image}:{version}".format(
+                cmd = "docker run -d -p {port_map} --network {net} --name {name} {image}:{version}".format(
```

Comment:
It would be cleaner to use a Python Docker API instead of calling the `docker` executable.

Reply:
The idea was to keep it close to bash, because it is unclear if python is the way to go forward.